scripts/debug-precompile.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Do we need this script in develop ?

Reply:
This script is used to deploy `ComplianceProxy` contract on the network so we can pass `ComplianceBridge` unit test.
It's kind of debugging script that is used only for internal development.